scripts/datasets/ade20k.py

```diff
@@ -38,5 +38,4 @@ def download_ade(path, overwrite=False):
             os.remove(_TARGET_DIR)
         # make symlink
         os.symlink(args.download_dir, _TARGET_DIR)
-    else:
```

Comment:
I don't understand this part. If the file is already downloaded locally, user can link it. Otherwise, gluoncv will download the dataset.

Reply:
`python ade20k.py  --download-dir /data/ade20`

Currently only create a symlink